### PR TITLE
Finished inserting waiver

### LINKED ISSUE #92: Display Waiver on the WaiverPage
Right now the Waiver pdf is not shown on the waiver page. For this task, add the Waiver component to the WaiverPage so it looks like this [Figma mockup ](https://www.figma.com/file/EIsY92RASe05zbSAfjDHcI/cityfarmslo-user-highfidel-v4?node-id=3%3A1065).

### PR DESCRIPTION
**Pull request by adellevo**

### CODE REVIEW

File: frontend/src/components/WaiverComponent/Waiver.jsx [hunk 1]
```diff
@@ -6,25 +6,25 @@ import releaseWaiver from "./WAIVER FORM 2021.docx.pdf";
...
-    width: 94.27vh !important;
-    height: 122vh !important;
-  }
-`;
+// const StyledPage = styled(Page)`
+//   & .react-pdf__Page__canvas {
+//     width: 94.27vh !important;
+//     height: 122vh !important;
+//   }
+// `;
```
Comment (vwinstea) [inline frontend/src/components/WaiverComponent/Waiver.jsx]: Feel free to delete instead of commenting out

**Review by vwinstea — APPROVED** [review]
Looks good! The bottom of the waiver is cut off on my screen, but I can look into that if you don't have the time.

File: frontend/src/components/WaiverComponent/Waiver.jsx (hunk 1 above)
Comment (adellevo) [reply frontend/src/components/WaiverComponent/Waiver.jsx]: Forgot to do that earlier, but I just went through and removed the comments from the WaiverPage and Waiver files

**Comment by adellevo** [comment]
> Looks good! The bottom of the waiver is cut off on my screen, but I can look into that if you don't have the time.

Ah I do have time, but I think it'd probably be better if you look into it bc idk how to reproduce that error for testing. I have `overflow-y:scroll` in the InnerWaiver component in the Waiver.jsx file, but I wasn't able to test that out since it shows up fine for me. Guess it doesn't work though 😅
Screen Shot 2022-03-29 at 2 05 51 AM

**Comment by vwinstea** [comment]
I tried fixing it but I'm still running into that cutoff issue so I think I'll merge this branch in now and then try to fix it after the meeting.

**Pull request merged**